.github/workflows/arm-kernel-images.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and release kernel images (ARM)
+
+on:
+  pull_request:
+    paths:
+      - 'experimental/arm/kernel/**'
+    branches: [main]
+  push:
+    paths:
+      - 'experimental/arm/kernel/**'
+    branches: [main]
+
+defaults:
+  run:
+    working-directory: experimental/arm/kernel
+
+jobs:
+  # build:
+  #   if: github.event_name != 'push'
+  #   runs-on: ubuntu-latest
+  #   permissions:
+  #     contents: read
+  #   strategy:
+  #     fail-fast: false
+  #     matrix:
+  #       version: [ 5.10.77 ]
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - name: Build kernel
+  #       run: make
+  release:
+    if: github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ 5.10.77 ]
+    env:
+      KERNEL_VERSIONS: ${{ matrix.version }}
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
note: there is a `checkout@v3`, some of my pet project jobs started to throw notices about it. In these cases it's only a `v2` to `v3` replace.

(yes copy paste)

Reply:
https://github.com/weaveworks-liquidmetal/image-builder/issues/46

Reply:
uuugh but then i have to dismiss all your reviews 😂

Reply:
that's why I created a new issue for that ;)

Reply:
noice ;)